dask_mpi/tests/basic.py

```diff
@@ -0,0 +1,27 @@
+import sys
+
+from time import sleep
+from distributed import Client
+from distributed.metrics import time
+from tornado import gen
+
+from dask_mpi.core import initialize
+
+scheduler_file = sys.argv[1]
+
+initialize(scheduler_file=scheduler_file)
+
+with Client(scheduler_file=scheduler_file) as c:
```

Comment:
As above, I'm interested in making this `scheduler_file=` keyword unnecessary

Reply:
Agreed.

Reply:
This has now been removed from the `initialize` function with commit https://github.com/dask/dask-mpi/pull/6/commits/fd335be0bfab4ec67cfe58b749bc3b2facbcd221.